coredb-operator/src/statefulset.rs

```diff
@@ -53,6 +56,74 @@ pub fn stateful_set_from_cdb(cdb: &CoreDB) -> StatefulSet {
             ..VolumeMount::default()
         },
     ]);
+    let mut containers = vec![
```

Comment:
move containers into separate variable in order to enable conditionally appending the sidecar